src/xml2js.coffee

```diff
@@ -169,6 +169,8 @@ class exports.Builder
             if typeof child is 'string' && @options.cdata && requiresCDATA child
               element = element.ele(key).raw(wrapCDATA child).up()
             else
+              if child === undefined || child === null
```

Comment:
My coffescript skills aren't the best.

http://coffeescript.org/#try:child%20%3D%20''%0A%0Aif%20child%20%3D%3D%20undefined%20%7C%7C%20child%3F%0A%20%20%20%20child%20%3D%20'asdf'

That doesn't look right, any advice?